packages/components/icons/src/fixtures/CustomIconReact.tsx

```diff
@@ -0,0 +1,37 @@
+const SvgCustomIcon = () => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width={35}
+    height={35}
```

Comment:
I think it would be better for this icon to not have size attributes.
I believe the intention is that the size of the `svg` is controlled by the icon size so it grows or shrinks with it.

Reply:
This is meant to be an example of a non-standard "custom" svg that a consumer of the library would pass to this component.

To me "custom" means that we do not override the properties that the user specified. If the user wanted the svg to scale, all they would have to do is remove the `width` and `height` from the svg.

Reply:
I can remove the width/height here if you'd prefer that for the docs examples though

Reply:
Thanks for sharing your reasoning, Byron 👍 

The way I see it, the main point of this component is to make sure the icons size is consistent across applications, not matter if they are from `ui-kit` or custom. I actually think we should update the description of the component to better explain about this.

If users provide a custom icon with a specific size, we won't get the expected behaviour because, as in the current state with the fixture svg, making the `CustomIcon` component bigger does not make the svg bigger, which I think that's what we want.

So yes, I would suggest to remove the size properties of the fixture SVG and also to update the description of the component (we can ask Filip for help).